docs/configuration.md

```diff
@@ -202,6 +202,15 @@ of the most common options to set are:
   </td>
   <td>2.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.driver.minMemoryOverhead</code></td>
+  <td>None</td>
+  <td>
+    The minimum amount of non-heap memory to be allocated per driver process in cluster mode, in MiB unless otherwise specified, if <code>spark.driver.memoryOverhead</code> is not defined.
+    This option is currently supported on YARN and Kubernetes.
+  </td>
+  <td>3.5.2</td>
```

Comment:
What value should be added here? I could not find a roadmap of future releases for spark, but saw the latest release is '3.5.1'

Reply:
oops, answered this in a different comment, should be 4.0.0